Apps/Patient/src/Services/IPatientDataService.cs

```diff
@@ -162,6 +162,35 @@ public record DiagnosticImagingExam : PatientData
         public override string Type { get; } = nameof(DiagnosticImagingExam);
     }
 
+    /// <summary>
+    /// BC Cancer screening exam patient data.
+    /// </summary>
+    public record BcCancerScreeningExam : PatientData
+    {
+        /// <summary>
+        /// Gets or sets the program name.
+        /// </summary>
+        public string? ProgramName { get; set; }
+
+        /// <summary>
+        /// Gets or sets the file id.
+        /// </summary>
+        public string? FileId { get; set; }
+
+        /// <summary>
+        /// Gets or sets the event datetime.
+        /// </summary>
+        public DateTime EventDateTime { get; set; }
+
+        /// <summary>
+        /// Gets or sets the result datetime.
+        /// </summary>
+        public DateTime ResultDateTime { get; set; }
+
+        /// <inheritdoc/>
+        public override string Type { get; } = nameof(BcCancerScreeningExam);
```

Comment:
I'm no longer sure I like the idea of including "Exam" in the property (or class name itself) since we're now planning to use this for both results & recalls.  I hadn't realized it was being passed to the front-end.  I'd suggest renaming to `BcCancerScreening` or `BcCancerScreeningEvent`.